packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -262,22 +284,6 @@ export class MessageWidget extends Component {
     }
   }
 
-
-  /**
-   * Update scroll position when activity list changes
-   * @param {object} firstActivity
-   * @param {object} previousFirstActivity
-   * @param {number} prevScrollTop
-   * @returns {null}
-   */
-  updateScroll(firstActivity, previousFirstActivity, prevScrollTop) {
-    const {activityList} = this;
-
-    if (firstActivity && previousFirstActivity && firstActivity.id !== previousFirstActivity.id) {
-      activityList.setScrollTop(activityList.getScrollHeight() - this.scrollHeight + prevScrollTop);
-    }
-  }
```

Comment:
Yea, can't get rid of this because it breaks message loading behavior.

Reply:
Darn, I was hoping to simplify the scroll tracking behavior. That stinks.